11LessonEleven/homework/css/inspired.css

```diff
@@ -0,0 +1,74 @@
+
+
+/* Laptop, Default size  */
+.box{
+  width: 40%;
+  height: 150px;
+  text-align: left;
+  color: black;
+  float: left;
+  padding-left: 5%;
+  background-color: #F7F7F7;
```

Comment:
I got it, thanks for you feedback.


On Jun 22, 2016 8:40 PM, "Victoria Chuang" <notifications@github.com> wrote:

> In 11LessonEleven/homework/css/inspired.css
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/15#discussion_r68164437>
> :
>
> > @@ -0,0 +1,74 @@
> > +
> > +
> > +/* Laptop, Default size  */
> > +.box{
> > +  width: 40%;
> > +  height: 150px;
> > +  text-align: left;
> > +  color: black;
> > +  float: left;
> > +  padding-left: 5%;
> > +  background-color: #F7F7F7;
>
> be careful! if you don't have the opacity, this color will cover the
> background image of its outer div. instead of a hex code, use rgba(value,
> value, value, value) to control the opacity with a.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/15/files/1463fa9ae6af4741e4a288de1da8708db8201ba9#r68164437>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASRWl7phl20jkZcihmJv4K7pzOO5l7Daks5qOeQIgaJpZM4IV3fY>
> .
>